Makefile

```diff
@@ -597,7 +597,7 @@ check-symbols: startup_files libc
 
 	# Check that the computed metadata matches the expected metadata.
 	# This ignores whitespace because on Windows the output has CRLF line endings.
-	diff -wur "$(CURDIR)/expected/$(MULTIARCH_TRIPLE)" "$(SYSROOT_SHARE)"
+	diff -wur "$(CURDIR)/expected/$(MULTIARCH_TRIPLE)/$(THREAD_MODEL)" "$(SYSROOT_SHARE)"
```

Comment:
I'm not sure this change in necessary, or at least we wouldn't want the default build to install into single.

I'm hoping that eventually we can just ship a single copy of libc since modern engines can handler lowering atomic accesses to non-atomics ones, and the linker can lower TLS to regular static data.

Reply:
I'm not sure what you're proposing... is there some other way in the current state of the PR to have all the same symbols for both thread models?